### DISCUSSION TITLE
Is there an alert object available to create a new alert on a panel

### DISCUSSION THREAD

**Discussion by abhishek-1985** [grafana/grafonnet#101] [Q&A] [answered]
Hi,

Am looking for an example how to leverage an alert object to create a new alert tied to a panel. Going through the docs , couldn't find APIs available for alert . Can someone help me with example if `with` statements if its supported ?

**Comment by malcolmholmes** [comment, marked as answer]
It is only in legacy alerts that they are created inside dashboards. This is no-longer the case. Grafonnet does not, at this time, support alerts.
